### PR TITLE
Remove tabs if followed by new lines (or vice versa)

### PR DESCRIPTION
**Pull request by annando**
Sometimes there can be tabs in the BBCode. We can safely remove them when new lines are around. Same goes with line feeds.

### CODE REVIEW

**Comment by nupplaphil** [comment]
New/adapting test? :-X .. :-)

**Comment by annando** [comment]
Too late for today :sleeping_bed:

**Comment by MrPetovan** [comment]
Nothing to do for tests until I remove the `system.remove_duplicated_lines` config key next release. The changed code isn’t currently hit by the tests because of the default value for this key.

File: src/Content/Text/BBCode.php [hunk 1]
```diff
@@ -1338,11 +1338,11 @@ public static function convert(string $text = null, bool $try_oembed = true, int
 				// removing multiplicated newlines
 				if (DI::config()->get('system', 'remove_multiplicated_lines')) {
 					$search = [
-						"\n\n\n", "\n ", " \n", "[/quote]\n\n", "\n[/quote]", "[/li]\n", "\n[li]", "\n[*]", "\n[ul]", "[/ul]\n", "\n\n[share ", "[/attachment]\n",
+						"\n\n\n", "\n ", " \n", "\n\t", "\t\n", "\n\r", "\r\n", "[/quote]\n\n", "\n[/quote]", "[/li]\n", "\n[li]", "\n[*]", "\n[ul]", "[/ul]\n", "\n\n[share ", "[/attachment]\n",
```
Comment (MrPetovan) [inline src/Content/Text/BBCode.php]: The CRLF conversion to LF is already done on line 1273.
```suggestion
						"\n\n\n", "\n ", " \n", "\n\t", "\t\n", "\n\r", "[/quote]\n\n", "\n[/quote]", "[/li]\n", "\n[li]", "\n[*]", "\n[ul]", "[/ul]\n", "\n\n[share ", "[/attachment]\n",
```

File: src/Content/Text/BBCode.php [hunk 2]
```diff
@@ -1338,11 +1338,11 @@ public static function convert(string $text = null, bool $try_oembed = true, int
...
 						"\n[h1]", "[/h1]\n", "\n[h2]", "[/h2]\n", "\n[h3]", "[/h3]\n", "\n[h4]", "[/h4]\n", "\n[h5]", "[/h5]\n", "\n[h6]", "[/h6]\n"
 					];
 					$replace = [
-						"\n\n", "\n", "\n", "[/quote]\n", "[/quote]", "[/li]", "[li]", "[*]", "[ul]", "[/ul]", "\n[share ", "[/attachment]",
+						"\n\n", "\n", "\n", "\n", "\n", "\n", "\n", "[/quote]\n", "[/quote]", "[/li]", "[li]", "[*]", "[ul]", "[/ul]", "\n[share ", "[/attachment]",
```
Comment (MrPetovan) [inline src/Content/Text/BBCode.php]:
```suggestion
						"\n\n", "\n", "\n", "\n", "\n", "\n", "[/quote]\n", "[/quote]", "[/li]", "[li]", "[*]", "[ul]", "[/ul]", "\n[share ", "[/attachment]",
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by annando** [comment]
Test is added

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**